cypress/e2e/invoices/approve-invoice-is-disabled.cy.js

```diff
@@ -0,0 +1,122 @@
+import { DevTeams, TestTypes, Permissions } from '../../support/dictionary';
+import { Invoices, InvoiceView } from '../../support/fragments/invoices';
+import { Budgets } from '../../support/fragments/finance';
+import Approvals from '../../support/fragments/settings/invoices/approvals';
+import TopMenu from '../../support/fragments/topMenu';
+import Organizations from '../../support/fragments/organizations/organizations';
+import Users from '../../support/fragments/users/users';
+import { NewOrder, BasicOrderLine, Orders } from '../../support/fragments/orders';
+import NewOrganization from '../../support/fragments/organizations/newOrganization';
+import { INVOICE_STATUSES } from '../../support/constants';
+
+describe('Invoices', () => {
+  const organization = NewOrganization.getDefaultOrganization();
+  const testData = {
+    organization,
+    order: {},
+    invoice: {},
+    user: {},
+  };
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      const { fiscalYear, fund, budget } = Budgets.createBudgetWithFundLedgerAndFYViaApi();
+
+      testData.fiscalYear = fiscalYear;
+      testData.fund = fund;
+      testData.budget = budget;
+
+      Organizations.createOrganizationViaApi(testData.organization);
+    });
+
+    cy.createTempUser([
+      Permissions.uiInvoicesApproveInvoices.gui,
+      Permissions.viewEditCreateInvoiceInvoiceLine.gui,
+      Permissions.viewEditDeleteInvoiceInvoiceLine.gui,
+      Permissions.uiInvoicesPayInvoices.gui,
+    ]).then((userProperties) => {
+      testData.user = userProperties;
+    });
+  });
+
+  beforeEach(() => {
+    testData.order = NewOrder.getDefaultOrder({ vendorId: organization.id });
+    testData.orderLine = BasicOrderLine.getDefaultOrderLine({
+      listUnitPrice: 98,
+      fundDistribution: [{ code: testData.fund.code, fundId: testData.fund.id, value: 100 }],
+    });
+
+    Orders.createOrderWithOrderLineViaApi(testData.order, testData.orderLine).then((order) => {
+      testData.order = order;
+
+      Invoices.createInvoiceViaApi({
+        vendorId: testData.organization.id,
+        accountingCode: testData.organization.erpCode,
+      }).then((invoice) => {
+        testData.invoice = invoice;
+      });
+    });
+
+    cy.login(testData.user.username, testData.user.password, {
+      path: TopMenu.invoicesPath,
+      waiter: Invoices.waitLoading,
+    });
+  });
+
+  after('Delete test data', () => {
+    Organizations.deleteOrganizationViaApi(testData.organization.id);
+    Users.deleteViaApi(testData.user.userId);
```

Comment:
Probably need to delete created Orders and Invoices

Reply:
System does not allow to delete any data related to transaction